modules/registry-configuring-registry-storage-swift-trust.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * registry/installing-openstack- .adoc
+// * registry/configuring-registry-operator.adoc
+// * registry/configuring-registry-storage-openstack-user-infrastructure.adoc
+
+[id="registry-configuring-registry-storage-swift-trust_{context}"]
+= Configuring the Image Registry Operator to trust Swift storage
+
+You must configure the Image Registry Operator to trust {rh-openstack-first} Swift storage.
+
+// to allow the client to pull the image layers from the image registry rather than from links directly from Swift.
+
+.Procedure
+
+* To configure the Image Registry Operator to trust OpenStack Swift storage, in the `config.imageregistry` object, change the value of the `spec.disableRedirect` field to `true`.
+
++
+[source,terminal]
+----
+$ oc patch configs.imageregistry.operator.openshift.io cluster --type merge --patch '{"spec":{"disableRedirect":"true"}}'
```

Comment:
The cmd shoud be, true value is a boolean 
oc patch configs.imageregistry.operator.openshift.io cluster --type merge --patch '{"spec":{"disableRedirect":true}}'

Reply:
```suggestion
$ oc patch configs.imageregistry.operator.openshift.io cluster --type merge --patch '{"spec":{"disableRedirect":true}}'
```

@xiuwang All good if this is committed?

Reply:
Yes, other LGTM